dap_prinz_green_jobs/analysis/ojo_analysis/occupation_similarity.py

```diff
@@ -0,0 +1,258 @@
+"""
+Get counts and proportions of skills in each occupation.
+Use these to create occupation skill similarity measures.
+"""
+
+import dap_prinz_green_jobs.analysis.ojo_analysis.process_ojo_green_measures as pg
+from dap_prinz_green_jobs.getters.data_getters import save_to_s3, load_s3_data
+from dap_prinz_green_jobs import BUCKET_NAME, PROJECT_DIR, logger, analysis_config
+from dap_prinz_green_jobs.getters.ojo_getters import (
+    get_mixed_ojo_location_sample,
+    get_mixed_ojo_salaries_sample,
+    get_large_ojo_location_sample,
+    get_large_ojo_salaries_sample,
+)
+
+from datetime import datetime
+import yaml
+import os
+
+import pandas as pd
+import numpy as np
+from sklearn.metrics.pairwise import cosine_similarity
+from tqdm import tqdm
+
+
+def find_skill_similarity(
+    occ_skills_info: dict,
+    esco_id_2_name: dict,
+    green_esco_id: list,
+    occ_agg_data: str = "outputs/data/ojo_application/extracted_green_measures/analysis/occupation_aggregated_data_20231220_all.csv",
```

Comment:
would use the date from the analysis config i.e. `analysis_config["occ_date] (i forget exactly how to call it but you get the gist)` just in case it changes again